doc/terms_x-road_docs.md

```diff
@@ -20,23 +20,23 @@ Document ID: TA-TERMS
 - [2 Participants of X-Road](#2-participants-of-x-road)
 - [3 Trust services](#3-trust-services)
 - [4 Roles of X-Road member](#4-roles-of-x-road-member)
-  * [4.1 In terms of dataservice](#41-in-terms-of-dataservice)
-  * [4.2 In terms of management of security server](#42-in-terms-of-management-of-security-server)
+* [4.1 In terms of dataservice](#41-in-terms-of-dataservice)
```

Comment:
Subsection-headings must be indented. Please restore the original indentation. This comment is applicable to subsection headings 4.x, 6.x and 7.x.